core/common/src/test/FrustumPlanes.test.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import { FrustumPlanes } from "../geometry/FrustumPlanes";
+import { Frustum } from "../Frustum";
+import { Point3d } from "@itwin/core-geometry";
+
+describe("FrustumPlanes", () => {
```

Comment:
Also test how it handles a bad frustum (e.g., input points do not define a valid frustum)?

Reply:
done